src/main/scala/se/nimsa/dicom/Value.scala

```diff
@@ -0,0 +1,52 @@
+package se.nimsa.dicom
+
+import java.text.{ParsePosition, SimpleDateFormat}
+import java.util.Date
+
+import akka.util.ByteString
+import se.nimsa.dicom.VR.VR
+
+/**
+  * Utility methods for parsing DICOM attribute value bytes to strings, numbers, dates, etc.
+  */
+object Value {
+
+  final lazy val multiValueDelimiter = "\\"
```

Comment:
IMO it's more readable if triple quotes are used.

Reply:
Good point, I'll make the change